src/css/categories.css

```diff
@@ -0,0 +1,38 @@
+.product-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+    gap: 20px; /* Spacing between grid items */
+    justify-content: center; /* Center horizontally */
+    align-items: center; /* Center vertically */
+    margin-left: 15%;
+    margin-right: 15%;
+}
+  
+.product-card {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #ccc;
+    padding: 10px;
+    text-align: center;
+    width: 200px;
+    min-height: 300px;
+    justify-content: center; /* Center horizontally */
+    align-items: center; /* Center vertically */
+}
+
+.product-card img {
+    width: 80%; /* Adjust the width as needed */
+    height: auto; /* Allow the height to adjust proportionally */
+}
+
```

Comment:
Finally, having the price in the same spot is important imho. You can play with this later. A quick & dirty way would be adding `position: relative` on .product-card and then adding:
```css
.product-card p {
  position: absolute;
  bottom: 0px;
}
```